docs/howto-use-applicationinsights-metrics.md

```diff
@@ -0,0 +1,123 @@
+# How to use Application Insights metrics
```

Comment:
This doc should probably go into a more broader "dev-guide.md", where we add more info on how to extend the platform